### PR TITLE
Explain how link is used for BLOCK_CREATE

### PR DESCRIPTION
**Pull request by RickiNano**

### CODE REVIEW

File: docs/commands/rpc-protocol.md
Comment (zhyatt) [inline docs/commands/rpc-protocol.md:612]: @dsiganos Can you confirm if this optional `destination` for `block_create` RPC is supposed to be the account public key or the account address `nano_`? That is unclear and may need to be updated.

**Review by zhyatt — CHANGES_REQUESTED** [review]
Requesting some updates/details here. There is a related PR that is waiting on some clarity as well: https://github.com/nanocurrency/nano-docs/pull/561

File: docs/commands/rpc-protocol.md
Comment (RickiNano) [reply docs/commands/rpc-protocol.md]: Good suggestions. I have updated the pr

Comment (zhyatt) [reply docs/commands/rpc-protocol.md]: Thanks! Pushed a small link fix that was removed by GitHub in my comment above. Will wait for feedback on the other item before merging.

File: docs/commands/rpc-protocol.md
Comment (zhyatt) [reply docs/commands/rpc-protocol.md]: Thanks @dsiganos I added links to the glossary to clarify this

**Pull request merged**
